libs/net/dal/Services/AVOverviewInstanceService.cs

```diff
@@ -88,20 +88,21 @@ public override AVOverviewInstance Update(AVOverviewInstance entity)
             }
             else
             {
-                var originalItems = this.Context.AVOverviewSectionItems.Where(s => s.SectionId == section.Id).ToArray();
+                var originalItems = this.Context.AVOverviewSectionItems.AsNoTracking().Where(s => s.SectionId == section.Id).ToArray();
                 originalItems.Except(section.Items).ForEach(s =>
                 {
                     this.Context.Entry(s).State = EntityState.Deleted;
                 });
-                this.Context.Update(section);
+                this.Context.Entry(section).State = EntityState.Modified;
```

Comment:
this.Context.Update(section) should change the state to Modified. Why does this need to change?

Reply:
Good question, but for some reason this line seems to be the cause of the issue, which means the removed record cannot actually be deleted.